package.json

```diff
@@ -15,14 +15,14 @@
     "copy-concurrently": "^1.0.0",
     "aproba": "^1.1.1",
     "fs-write-stream-atomic": "^1.0.8",
-    "mkdirp": "^0.5.1",
+    "mkdirp": "^1.0.4",
```

Comment:
`mkdirp` isn't even used in the package. It can just be removed. See my PR from 2018: #2